src/main/java/Duke.java

```diff
@@ -28,11 +28,11 @@ private static void load() throws IOException, DukeException {
             switch(type) {
```

Comment:
Should have a space before the bracket i.e `switch (type)`. Just a minor code style thing though